build/ci/dh/api-dh/ci.appsettings.Development.json

```diff
@@ -8,7 +8,8 @@
   },
   "ApiClientSettings": {
     "MeteringPointBaseUrl": "https://app-webapi-mpt-u-001.azurewebsites.net",
-    "ChargesBaseUrl": "https://app-webapi-charges-u-001.azurewebsites.net"
+    "ChargesBaseUrl": "https://app-webapi-charges-u-001.azurewebsites.net",
+    "WholesaleBaseUrl": "https://app-webapi-wholsal-u-001.azurewebsites.net"
   },
   "B2C_TENANT_ID" : "4a7411ea-ac71-4b63-9647-b8bd4c5a20e0",
```

Comment:
Im not sure but is it ok exposing these values?

Reply:
@lasrinnil I agree, and the values can be fetched from key vault secrets

Reply:
@djorgensendk could you manage this in MD? It's based on what has already been done and seems to be based on thoughts by Texas Rangers (maintaining and copying this file is described in the repo gettting started documentation).